pkg/datastore/phpapi/phpapi.go

```diff
@@ -31,7 +31,7 @@ type Specification struct {
 	Server string
 }
 
-func NewPHPAPI(apitoken string) (*PHPAPI, error) {
+func NewPHPAPI(apitoken string) (datastore.Datastore, error) {
```

Comment:
Why change the return type?

Reply:
No need to expose the internal type but more so the compiler will verify that `PHPAPI` implements `datastore.Datastore` here as well at call sites. But it's not that important - I can change it back if you want.

Reply:
Oh, I read it the other way round, so I thought this would remove one level of type-checking ;-). So the new variant is better, thanks for the explanation!

Reply:
@Oats87 also said we are free to kill the php api bits

Reply:
Ok.  I'll do that later in a follow-up.